.ruby-version

```diff
@@ -1 +1 @@
-2.3.0
+2.3.1
```

Comment:
Oh, I didn't realize vets-api was still so out of date on ruby version. It really shouldn't matter.